src/protocol/prss.rs

```diff
@@ -449,7 +479,7 @@ pub mod test {
     #[test]
     fn three_party_values() {
         const IDX: u128 = 7;
-        let [p1, p2, p3] = make_participants();
+        let [p1, p2, p3] = ParticipantSetup::default().into_participants();
```

Comment:
Maybe you can make a wrapper function for this file, so that you don't need to change all those lines.